src/adminEntity.ts

```diff
@@ -5,10 +5,12 @@ import DefaultAdminSite from './adminSite'
 import ManyToManyWidget from './widgets/manyToManyWidget'
 import InvalidDisplayFieldsException from './exceptions/invalidDisplayFields.exception'
 import { WidgetConstructor } from './widgets/widget.interface'
+import { Request, Response } from 'express'
 
 abstract class AdminEntity {
   abstract entity: EntityType
   listDisplay: string[] | null = null
+  listActions: Array<{ label: string; action: (request: Request, response: Response) => void }>
```

Comment:
Extract this type into a `ListAction` type, to be exported from the library. This type will then be `ListAction[]`